src/results/metadata.jl

```diff
@@ -111,3 +116,34 @@ function get_relative_cover(name::String, timestep::Union{Int64, UnitRange{Int64
     # this syntax prevents auto-rename to relative_cover_mean
     return combine(groupby(table, :UNIQUE_ID), :relative_cover => mean => :relative_cover)
 end
+
+function setup_result_metadata_routes()
```

Comment:
In the ReefGuide API, separate setup functions needed to be defined on server start because it was reliant on a global variable that gets created on server start up.

I'm not sure this is the case here? The routes could be pre-defined without issue?

The reason this is semi-important is that the routes could potentially be pre-compiled reducing package start up times. Currently, Julia will need to compile these during runtime.